source/security/mqtt-tls.rst

```diff
@@ -0,0 +1,68 @@
+MQTT-TLS
+==============
+
+This document describes how to configure dojot to use MQTT over TLS.
+
+.. contents:: Table of Contents
+  :local:
+
+tl;dr
+---------
+
+For a device to connect using TLS with Mosquitto, it must posses:
+  - A key pair (.key file)
+  - A certificate signed by a Certificate Authority (CA) trusted by Mosquitto (.crt file)
+  - The certificate of this CA (.crt file)
+  - An entry on Mosquitto Access Control List (ACL), allowing the device to publish on a specific topic.
+  - (optional and not so necessary) A Certificate Revocation List (CRL)
+
+
+When a device is created, Device-manager will (automatic) configure the following components:
+  - IoTAgent -> will registry the new device on its internal cache.
+  - MQTT-Manager -> Will create an entry on the ACL. Allowing the device to publish on a specific topic.
```

Comment:
Perhaps:

MQTT-Manager -> Will create an entry on the ACL, allowing the device to publish on a specific topic.

Reply:
Ok